cypress/integration/publishingFlows/postBrowserPersistence.spec.js

```diff
@@ -0,0 +1,209 @@
+describe('Post Editor', () => {
+  function getPostContent() {
+    return cy.get('@articleForm').findByLabelText(/^Post Content$/i);
+  }
+
+  describe('v1 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
+      });
+    });
+
+    describe(`revert changes`, () => {
```

Comment:
Seems both of these are failing the output is close but slightly different in each case.